app/controllers/wizard/steps/country_of_origin_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Wizard
+  module Steps
+    class CountryOfOriginController < BaseController
+      def show
+        @country = Wizard::Steps::CountryOfOrigin.new(session)
```

Comment:
How do you feel about renaming this to step? Feels like it wouldn't collide and also highlights we're coding to an interface and not an implementation, etc